week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -14,7 +14,33 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
+let newArr=[];
+
 function sortAges(arr) {
+  for ( let i=0; i<arr.length; i++){
+    if (typeof arr[i] === "number"){
+      newArr.push(arr[i])
+    }
+  }
+  console.log(newArr);
+
+  for(let  i=0;i<newArr.length;i++) //Hold the first element
+  {
+      for(let j=i+1;j<newArr.length;j++) //Hold the next element from the first element
+      {
+          if(newArr[i] > newArr[j]) //comparing first and next element
+          {
+              let tempValue = newArr[j];   
+              newArr[j] = newArr[i];
+              newArr[i] = tempValue;
+          }
+      }
+  }
+

```

Comment:
Try using `map` and `sort` here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
Task says:  (without using the built-in javascript
sort method 😎)

Reply:
Thanks! I should have read it first 😸